src/app/auth/index.types.d.ts

```diff
@@ -2,3 +2,23 @@ type NewAccountWithPhoneIdentifier = {
   kratosUserId: UserId
   phone: PhoneNumber
 }
+
+type LoginDeviceUpgradeWithPhoneResult = {
+  success: true
+  sessionToken?: SessionToken
+}
+
+type initiateTotpRegistrationResult = {
+  flow: FlowId
```

Comment:
Can we have more granular / better naming around flow id. I don't like that we are just conforming to kratos semantics here.